bin/nef-carbon

```diff
@@ -20,35 +20,68 @@ printHelp() {
     echo ""
     echo "${normal}$0 ${bold}--project ${normal}<project> ${bold}--output ${normal}<output> ${reset}"
     echo ""
+    echo "    Parameters"
+    echo ""
     echo "    ${lime}${bold}--project${reset}${normal} path where playground's pages are.${reset}"
     echo "    ${lime}${bold}--output${reset}${normal} path where Carbon snippets are saved to.${reset}"
+    echo ""
+    echo "    Options"
+    echo ""
+    echo "    ${lime}--background${reset} background color in hexadecimal."
+    echo "    ${lime}--theme${reset} carbon's theme."
+    echo "    ${lime}--size${reset} export file size [1-5]."
+    echo "    ${lime}--font${reset} carbon's font type."
+    echo "    ${lime}--lines${reset} shows/hides lines of code [true | false]."
+    echo "    ${lime}--watermark${reset} shows/hides the watermark [true | false]."
     echo "${reset}"
 }
 
 #: - Render
 
 ##
-#   generateCarbon(String projectPath, String outputPath)
+#   generateCarbon(String projectPath, String outputPath, background: String, theme: String, size: String, font: String, lines: String, watermark: String)
 #   - Parameter `project`: path to the project folder.
 #   - Parameter `outputPath`: path where Carbon snippets are saved to.
+#
+#   Options:
+#   - Parameter `background`: background color in hexadecimal. ex. 'bow'
```

Comment:
In hexadecimal, from the list, or both? The example contradicts the instructions.
```suggestion
#   - Parameter `background`: background color in hexadecimal or form a list of predefined colors. ex. 'bow'
```